### ISSUE TITLE
How to change three-finger swipes to four-finger

### ISSUE THREAD

**Issue by elais** [paperwm/PaperWM#333]
Three finger swipes aggravate my RSI and it's alot easier to swipe with four fingers than three. I've looked at the gestures.js file but am a little confused about where exactly three finger swipes are defined. Could you point me in the right direction?

**Comment by hedning** [comment]
When I first made the swipe support it wasn't possible to detect 4-finger swipes as gnome-shell already captured the events.

It's now possible to turn off gnome-shell's capture, so it should be fairly simple to add 4 finger support.

I think something like this might work, though there's a chance you'll have  to do something like this to make it sticky: `Main.wm._swipeTracker._touchpadGesture.connect(`notify::enabled`, () => { Main.wm._swipeTracker._touchpadGesture.enabled = false }`:

```diff
modified   gestures.js
@@ -56,7 +56,7 @@ function enable() {
      */
     signals.connect(stage, 'captured-event', (actor, event) => {
         if (event.type() !== Clutter.EventType.TOUCHPAD_SWIPE ||
-            event.get_touchpad_gesture_finger_count() > 3 ||
+            event.get_touchpad_gesture_finger_count() < 3 ||
             (Main.actionMode & Shell.ActionMode.OVERVIEW) > 0) {
             return Clutter.EVENT_PROPAGATE;
         }
modified   kludges.js
@@ -383,6 +383,9 @@ function init() {
     // Kill pinch gestures as they work pretty bad (especially when 3-finger swiping)
     registerOverrideProp(imports.ui.viewSelector, "PINCH_GESTURE_THRESHOLD", 0);
 
+    if (Main.wm._swipeTracker)
+        registerOverrideProp(Main.wm._swipeTracker._touchpadGesture, "enabled", false);
+
     registerOverridePrototype(Workspace.Workspace, '_isOverviewWindow', (win) => {
         let metaWindow = win.meta_window;
         return Scratch.isScratchWindow(metaWindow) && !metaWindow.skip_taskbar;
```

**Comment by hedning** [comment]
4a58b549e5b3d0ff46bb448d4509f4d4df87a067 should make 4-finger (in addition to 3-finger) swiping available for gnome-shell >= 3.36.0
